icons/clubforce.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Clubforce</title><path d="M1.251 0v13.325l3.977-3.017V3.902h8.54L18.992 0ZM22.75 2.43l-3.902 2.945v5.3l-3.608-2.5-3.386 2.5 5.963 4.123-5.89 4.417-5.742-4.343-3.092 2.43L11.926 24l10.823-7.877Z"/></svg>
```

Comment:
According to https://clubforce.com/media-centre/:

> Please do not modify the Clubforce logos

I'm wondering if we should add a background for the icon as we can see at page 10 from their [Brand Guidelines PDF](https://clubforce.com/app/uploads/2022/08/clubforce-brand-styleguide-1.pdf):
![image](https://github.com/simple-icons/simple-icons/assets/12817388/4fefa5c5-5bba-49ad-9304-a68ae2681005)

It is the only version we can find without wordmark.

Reply:
Unfortunately, I think you're right, @service-paradis. PR Updated.